media/css/firefox/whatsnew/whatsnew-132.scss

```diff
@@ -0,0 +1,129 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import "../../protocol/components/video";
+@import 'includes/mofo-donate-cta';
+
+.wnp-footer {
+    text-align: center;
+}
+
+.wnp-content-main {
+    background: #6863e5 linear-gradient(260deg, #ad35dc -30%, #6863e5 60%, #452789 110%);
+    color: $color-white;
+    padding-top: 50px;
+
+    .wnp-main-title {
+        @include text-title-xl;
```

Comment:
**Nit (optional):** The heading gets pretty cramped at 768px. I think this would benefit from having a different size declared for the 768-1024 break point.